src/studio/AppTemplates/AspNet/App/App.csproj

```diff
@@ -12,6 +12,7 @@
     <PackageReference Include="Altinn.App.Common" Version="3.2.1" />
```

Comment:
Is there something missing on line 11? The extra comment that I didn't like, but that was required?

Reply:
Yes, we still need it :(